src/select/common.ts

```diff
@@ -37,10 +39,11 @@ export const useTypeahead = ({
 
   const handleOnKeyPress = (e: React.KeyboardEvent) => {
     e.persist();
-    // skip the enter key
-    if (e.key === "Enter") return;
+    // skip all the keys eg: Enter, Alt, Shift
+    if (e.key.length > 1) return;
+
```

Comment:
onKeyPress isn't working with SelectDropdown so i'm using the onKeyDown event handler, but onkeydown event handler also captures Enter, Shift, Alt, etc etc key strokes so i'm skipping the keys which are longer than 1 character, this ensure we are not concatenating the `typed` string with Shift, Enter & other keywords.